### PR TITLE
[zscaler] Docs only integration tile

### PR DESCRIPTION
**Pull request by BoyangHuang**
### What does this PR do?

This PR is meant to implement the docs only integration tile z-scaler. It should include integration documentation and a dashboard json.

The logs pipeline PR is being built separately (facets, remappings), and internally.

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: zscaler/README.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+### Validation
+
+[Run the Agent's status subcommand][5] and look for `zscaler` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][6] for a list of metrics provided by this check.
```
Comment (maycmlee) [inline zscaler/README.md:118]: Should we say the integration doesn't include any metrics (the metadata.csv doesn't list any)?

File: zscaler/assets/dashboards/zscaler_overview.json [hunk 2]
```diff
@@ -0,0 +1 @@
...caler.tunneltype","@network.destination.ip","@network.client.ip","@network.client.port","@zscaler.rxbytes"],"show_date_column":true,"show_message_column":false,"message_display":"inline"},"layout":{"x":0,"y":2,"width":12,"height":3}}]},"layout":{"x":0,"y":18,"width":12,"height":6}}],"template_variables":[{"name":"ClientIP","default":"*","prefix":"@network.client.ip","available_values":[]},{"name":"DestinationIP","default":"*","prefix":"@network.destination.ip","available_values":[]},{"name":"UserID","default":"*","prefix":"@usr.id","available_values":[]},{"name":"HTTPHost","default":"*","prefix":"@http.url_details.host","available_values":[]},{"name":"HTTPProtocol","default":"*","prefix":"@zscaler.protocol","available_values":[]},{"name":"HTTPContentType","default":"*","prefix":"@zscaler.contenttype","available_values":[]},{"name":"ZscalerAppClass","default":"*","prefix":"@zscaler.appclass","available_values":[]},{"name":"Outcome","default":"*","prefix":"@evt.outcome","available_values":[]},{"name":"DNSRequestName","default":"*","prefix":"@zscaler.dns_req","available_values":[]}],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"tee-pdp-wqf"}
```
Comment (maycmlee) [inline zscaler/assets/dashboards/zscaler_overview.json:1]: Looks like the description hasn't been added.

File: zscaler/manifest.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Zscaler Integration",
+  "categories": [
+    ""
+  ],
+  "type": "check",
+  "is_public": false,
```
Comment (maycmlee) [inline zscaler/manifest.json:22]: "is_public" has to be set to "true" for the page to appear on the public docs integrations page.

**Review by sarah-witt — COMMENTED** [review]
Looks good, I left two small comments

File: zscaler/README.md (hunk 1 above)
Comment (BoyangHuang) [reply zscaler/README.md]: Good catch - updated this snippet in 6d0f6d3760e14bc5f33ba225b58cf2fdc189cc3e

File: zscaler/assets/dashboards/zscaler_overview.json (hunk 2 above)
Comment (BoyangHuang) [reply zscaler/assets/dashboards/zscaler_overview.json]: Ah, thank you. I copy-pasted the dashboard JSON from Zscaler's org, so this must've gotten lost in translation. Updated the title in 6d0f6d3760e14bc5f33ba225b58cf2fdc189cc3e

[the remark this answers is not in the document]
Comment (BoyangHuang) [reply zscaler/assets/dashboards/zscaler_overview.json]: Updated in 6d0f6d3760e14bc5f33ba225b58cf2fdc189cc3e!

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -123,6 +123,7 @@
 /vespa/                                  @gjoranv dd@vespa.ai
 /vns3/                                   @DataDog/agent-integrations
 /zabbix/                                 @KosukeKamiya
+/zscaler/                                @BoyangHuang @DataDog/agent-integrations
```
[the remark this answers is not in the document]
Comment (BoyangHuang) [reply .github/CODEOWNERS]: Updated in 6d0f6d3760e14bc5f33ba225b58cf2fdc189cc3e!

**Comment by BoyangHuang** [comment]
@sarah-witt @maycmlee Responded to your feedback! Thank you both for taking a look - it's my first integration-extras PR so I really appreciate it!

File: zscaler/README.md (hunk 1 above)
Comment (maycmlee) [reply zscaler/README.md]: Thanks! Can the metadata.csv file also be removed, since it's empty? Otherwise everything looks good!

Comment (maycmlee) [reply zscaler/README.md]: Thanks! Can the metadata.csv file also be removed, since it's empty? Otherwise everything looks good!

**Review by maycmlee — APPROVED** [review]
👍  from docs!

**Pull request merged**